### PR TITLE
Rewrites java docs into new format

### PR DESCRIPTION
**Pull request by ayip8**

### CODE REVIEW

File: docs/sdks/java.md [hunk 1]
```diff
@@ -31,136 +34,214 @@ There's an optional uber-jar with shaded and relocated guava and failsafe depend
...
-                                                 )
+prefabCloudClient.configClient().getContextStore().addContext(PrefabContext.newBuilder("User")
+                        .put("name", user.getName())
+                        .build());
 ```
 
-and globally or request-scoped with the ContextStore which will affect all logging, featureflag and config lookups.
+When global context is set, log levels and feature flags will evaluate in that context. Here are details on setting global context:
+
+<Tabs groupId="lang">
```
Comment (ayip8) [inline docs/sdks/java.md:119]: FYI these Micronout and Dropwizard instructions are copy pasted from the Context explanation page. I may DRY it up and use partials later.

File: docs/sdks/java.md [hunk 2]
```diff
@@ -31,136 +34,214 @@ There's an optional uber-jar with shaded and relocated guava and failsafe depend
...
+<TabItem value="micronaut" label="Micronaut">
+
+In your Prefab options, set your Prefab Context store to be your ServerRequestContextStore.
```
Comment (jkebinger) [inline docs/sdks/java.md:122]: This info about ServerRequestContextStore in particular is very micronaut specific so I'm not sure it should be top level

File: docs/sdks/java.md [hunk 3]
```diff
@@ -31,136 +34,214 @@ There's an optional uber-jar with shaded and relocated guava and failsafe depend
...
 
 ```java
+options.setContextStore(new ServerRequestContextStore());
```
Comment (jkebinger) [inline docs/sdks/java.md:125]: this is very micronaut specific so I don't know if needs the prominence it has here - maybe more of a caution like the ruby one for forking app servers

File: docs/sdks/java.md [hunk 4]
```diff
@@ -31,136 +34,214 @@ There's an optional uber-jar with shaded and relocated guava and failsafe depend
...
+          .put("email", user.email())
+          .build()
+      );
+```
+
+Learn more with the [Prefab + Micronaut example app](https://github.com/prefab-cloud/example-micronaut-app)
+
+</TabItem>
+
+<TabItem value="dropwizard" label="Dropwizard">
```
Comment (jkebinger) [inline docs/sdks/java.md:144]: I don't think we need the drop wizard vs micronaut things here - the snippet above isn't fully developed (we could copy paste more) compared to the one below , which may be over developed and too long, plus we can link to the full versions

**Pull request merged**